<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Admin Mode</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <widget class="QTableWidget" name="tableWidget_ShowAll">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>461</width>
             <height>231</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_ShowAllUsers">
           <property name="geometry">
            <rect>
             <x>15</x>
             <y>300</y>
             <width>141</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Show All Users</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <widget class="QTableWidget" name="tableWidget_ShowBy">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>40</y>
             <width>461</width>
             <height>231</height>
            </rect>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBox_UserId">
           <property name="geometry">
            <rect>
             <x>50</x>
             <y>320</y>
             <width>44</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_SearchById">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>280</y>
             <width>111</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Search by id</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_SearchByClass">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>280</y>
             <width>111</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Search by class</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_SearchByRole">
           <property name="geometry">
            <rect>
             <x>340</x>
             <y>280</y>
             <width>111</width>
             <height>25</height>
            </rect>
           </property>
           <property name="text">
            <string>Search by role</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="spinBox_ClassNumber">
           <property name="geometry">
            <rect>
             <x>200</x>
             <y>320</y>
             <width>44</width>
             <height>26</height>
            </rect>
           </property>
           <property name="maximum">
            <number>6</number>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox_Roles">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>320</y>
             <width>150</width>
             <height>25</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>50</number>
      </property>
      <property name="rightMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_AddStudent">
        <property name="text">
         <string>Add Student</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_AddTeacher">
        <property name="text">
         <string>Add Teacher</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_EditUser">
        <property name="text">
         <string>Edit User Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_DeleteUser">
        <property name="text">
         <string>Delete User</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
